### PR TITLE
Add token page

### PR DESCRIPTION
**Pull request by servatj**
Added new token page:

image
Added Token index page:

image

### CODE REVIEW

File: src/lib/explorer/TokenExplorer.svelte [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+            result.data = result.results;
+
+            callback(result);
+        }
+    };
+
+
+</script>
+
+<div class="trading-pairs">
```
Comment (miohtama) [inline src/lib/explorer/TokenExplorer.svelte:263]: Check class name

File: src/lib/explorer/TokenIndexExplorer.svelte [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+			if (filterJunk) {
+				params.filter_zero_volume = 'true';
+			}
+
+			if (chainSlug) {
+				params.chain_slug = chainSlug;
+			}
+
+			// https://tradingstrategy.ai/api/explorer/#/Pair/web_pairs
```
Comment (miohtama) [inline src/lib/explorer/TokenIndexExplorer.svelte:87]: Comment

File: src/routes/trading-view/[chain]/tokens/[token]/index.svelte [hunk 3]
```diff
@@ -0,0 +1,202 @@
+<script context="module">
+    /*
+        Render the pair trading page
```
Comment (miohtama) [inline src/routes/trading-view/[chain]/tokens/[token]/index.svelte:3]: Comment is incorrect

File: src/routes/trading-view/[chain]/tokens/index.svelte [hunk 4]
```diff
@@ -0,0 +1,85 @@
+<script context="module" lang="typescript">
+	/*
+
+		Render listing of all available exchanges
```
Comment (miohtama) [inline src/routes/trading-view/[chain]/tokens/index.svelte:4]: Comment is incorrect

**Comment by hieuh25** [comment]
Nitpicking: can the Tokens table have loading state instead of a default empty table?
image
Since some queries are fairly slow (> 5 seconds), the empty table makes the view feel like an error. 
The ultimate fix is to have pagination support from API but that can be improved later.

**Pull request merged**